bin/run-web-tool

```diff
@@ -1,3 +1,3 @@
 #! /bin/bash
 
-"$(pwd)"/bin/start-containter-with /bound/bin/run-r-scripts
+"$(pwd)"/bin/start-containter-with /bound/bin/run-r-scripts "${1:-TestPortfolio_Input}"
```

Comment:
This is the high level script what I expect users to call. It now has a sensible default to simplify testing.

Reply:
haven't test this myself yet, but if this works, great! 😄